config/initializers/statsd.rb

```diff
@@ -7,3 +7,12 @@
                  else
                    StatsD::Instrument::Backends::LoggerBackend.new(Rails.logger)
                  end
+
+# Initialize session controller metric counters at 0
+
+StatsD.increment(V0::SessionsController::STATSD_LOGIN_TOTAL_KEY, 0)
+StatsD.increment(V0::SessionsController::STATSD_LOGIN_FAILED_KEY, 0)
```

Comment:
Isn't this obsolete since we will never increment the `STATSD_LOGIN_FAILED_KEY` w/o a tag?  I suppose it can't hurt in case we do at a later date.